CONTRIBUTORS.md

```diff
@@ -0,0 +1,17 @@
+# Contributors
+
+- [@ENT8R](https://github.com/ENT8R)
+
+## Translators
+
+### German
+
+- [@rugk](https://github.com/rugk)
+
+### Spanish
+
+- David E. Barrera [@dbarrerap](https://github.com/dbarrerap)
+
+### Turkish
+
+- Ömür Turan [@omurturan](https://github.com/omurturan)
```

Comment:
Ah no, rather please add brackets around the GitHub handle. Also the same for `@dbarrerap` above. See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#contributors-file